stream_alert/rule_processor/handler.py

```diff
@@ -133,6 +160,69 @@ def get_alerts(self):
         """
         return self._alerts
 
+    def _send_to_firehose(self):
+        """Put all classified records into its respective Firehose Delivery Stream
+
+        Args:
+            categorized_payloads (defaultdict): Categorized set of payloads by classified type
+        """
+        def _chunk(record_list, chunk_size):
+            """Helper function to chunk payloads"""
+            for item in range(0, len(record_list), chunk_size):
+                yield record_list[item:item + chunk_size]
+
+        def _check_record_batch(batch):
+            """Helper function to verify record size"""
+            for index, record in enumerate(batch):
+                if len(str(record).encode('utf-8')) > MAX_RECORD_SIZE:
```

Comment:
Is the encoding here required? Please add a comment if so, since it's not clear and seems to be only used for the size check (the encoded string isn't saved).

Reply:
It's just to ensure there's no weird characters in the log file